### DISCUSSION TITLE
Announcing Nuemark!

### DISCUSSION THREAD

**Discussion by tipiirai** [nuejs/nue#159] [Announcements]
Nuemark is a Markdown-based text editing format for rich internet content. It places content at the heart of everything bringing (back) a content-first approach to web development. 

Check out the [announcement blog post](https://nuejs.org/blog/introducing-nuemark/)

<img src="https://nuejs.org/img/nuemark-content-big.png" width="500">

**Comment by seneca** [comment]
This is awesome. Thank you.

**Reply by tipiirai** [reply]
Great. Thanks!

**Comment by seneca** [comment]
Just noticed that ! video: heroic | nuemark in Creating rich, interactive content with Nuemark article is not showing du to maybe misplaced resource?

**Reply by tipiirai** [reply]
Now fixed. Thanks!

**Comment by indifferentghost** [comment]
Hey I noticed nuemark uses marked as the underlying library. Is there an easy way to add marked extensions for things like [footnotes](https://github.com/bent10/marked-extensions/tree/main/packages/footnote)?

**Reply by tipiirai** [reply]
Not right now, but this seems like a legit feature to have. Thanks for the tip.
